src/checks.rs

```diff
@@ -15,13 +15,17 @@ mod unordered_key;
 trait Check {
     fn run(&mut self, line: &LineEntry) -> Option<Warning>;
     fn name(&self) -> &str;
+    fn skip_comments(&self) -> bool {
+        true
```

Comment:
> Why this change is needed?
Yes, it was not obvious to me when I ran the tests.

The fact is that earlier comment lines were [previously skipped for checks](https://github.com/dotenv-linter/dotenv-linter/blob/e49d25a1d18f3519df49733ebe2a16a8a1a7d96b/src/checks.rs#L58).
But because this check was [out of the list](https://github.com/dotenv-linter/dotenv-linter/blob/e49d25a1d18f3519df49733ebe2a16a8a1a7d96b/src/checks.rs#L69), it was not applied for it.
We even have a [test for this](https://github.com/dotenv-linter/dotenv-linter/blob/e49d25a1d18f3519df49733ebe2a16a8a1a7d96b/tests/ending_blank_line.rs#L29).
I decided that taking it to the trait level would be a good decision, so a particular implementation can choose whether to consider comments.
Ready to discuss any suggestions :thinking:

Reply:
Thank you for explaining. I have forgotten about it 👍